backend/services/PullRequest.js

```diff
@@ -0,0 +1,67 @@
+const axios = require('axios')
+
+let pEndpoint = (year) => {
+  return `https://api.github.com/search/issues?q=is:pr created:${year}-10-01..${year}-10-31&sort=created&order=desc`
+}
+
+class PullRequest {
+  constructor (year) {
+    this.data = {}
+    this.year = year || (new Date()).getFullYear()
+  }
+
+  /**
+   * Retrieves all pull requests (github limits them to 100)
+   */
+  async getAll () {
+    try {
+      let response = await axios.get(pEndpoint(this.year))
+
+      this.data = response.data
+
+      return this
+    } catch (error) {
+      console.error(error)
+    }
+  }
+
+  /**
+   * Group pull requests by user id
+   */
+  groupByUser () {
+    let prArray = []
+
+    this.data.items.map((item) => {
```

Comment:
you only use `.map` if you want it to return a new array.

Otherwise, use `forEach`.